### PR TITLE
Rename `prefix_len ` to `prefix_length` which deprecated after v5.0.0

### PR DESCRIPTION
**Pull request by charleslxh**

### CODE REVIEW

File: lib/Elastica/Suggest/Term.php [hunk 1]
```diff
@@ -126,4 +126,16 @@ public function setMaxTermFrequency($max)
     {
         return $this->setParam('max_term_freq', $max);
     }
+
+    /**
+     * Set the number of distance which implementation to use for comparing how similar suggested terms are.
```
Comment (ruflin) [inline lib/Elastica/Suggest/Term.php]: Could we move this into a separate PR?

**Comment by ruflin** [comment]
@charleslxh Any update here? Would be nice to get this in.

**Comment by deguif** [comment]
@charleslxh @ruflin would you mind if I take care of this?

**Comment by ruflin** [comment]
@deguif Would be great if you could take this over.

File: lib/Elastica/Suggest/Term.php (hunk 1 above)
Comment (deguif) [reply lib/Elastica/Suggest/Term.php]: And this already exists but method is named `setStringDistanceAlgorithm`, probably added in the meantime.

**Comment by ruflin** [comment]
Closing this in favor of https://github.com/ruflin/Elastica/pull/1708
